### ISSUE TITLE
what's the meaning of meta files?

### ISSUE THREAD

**Issue by JINXvvv** [facebookresearch/supervision-by-registration#65]

**Comment by D-X-Y** [comment]
The files in meta contain both ground truth data and predicted labels. See https://github.com/facebookresearch/supervision-by-registration/blob/master/lib/xvision/evaluation_util.py#L40

**Comment by JINXvvv** [comment]
I'm sorry to trouble you again.I have seen the prediction and groundtruth attributes.But what confuses me is that I use the model in snapshots/CPM-SBR/checkpoint to test the testset data folder 114,the result is quite different from prediction results of the model in snapshots/CPM-SBR/meta.So which model should I use to test other video?

**Comment by D-X-Y** [comment]
It is a little bit strange. Results in meta file should be consistent with the predictions from the model loaded from checkpoint. Would you mind to provide more details? Such as your training script, checkpoint name, etc/

**Comment by JINXvvv** [comment]
my training script:lk_main.py --train_lists /data/glusterfs_cv/11103467/facetracking/lists/300VW/300VW.train.lst /data/glusterfs_cv/11103467/facetracking/lists/300W/300w.train.DET --eval_ilists /data/glusterfs_cv/11103467/facetracking/lists/300VW/300VW.test-1.lst --num_pts 68 --model_config ./configs/Detector.config --opt_config ./configs/LK.SGD.config --lk_config ./configs/mix.lk.config --video_parser x-1-1 --save_path /data/glusterfs_cv/11103467/facetracking/300VW_ckpt32 --init_model /data/glusterfs_cv/11103467/facetracking/ckpt32/checkpoint/cpm_vgg16-epoch-048-050.pth --workers 16 --pre_crop_expand 0.2 --sigma 4 --batch_size 16 --crop_perturb_max 5 --scale_prob 1 --scale_min 1 --scale_max 1 --scale_eval 1 --heatmap_type gaussian --print_freq 10

checkpoint name :cpm_vgg16-epoch-010-050.pth
meta name:eval-epoch-010-050-00-01.pth
When I run vis.py to visualize the prediction of eval-epoch-010-050-00-01.pth，the result is precise and stable.But when I run eval.py to test cpm_vgg16-epoch-010-050.pth,all the point are in a mess.Some points are even draw far away from the face.
![000026](https://user-images.githubusercontent.com/32635676/68727059-4bd05d00-05fe-11ea-8dd8-6ddecd9c67a8.png)

**Comment by JINXvvv** [comment]
很抱歉一直麻烦您，但是今天我仔细地debug了一下，确实两个模型相差结果太大了，而且300VW训练出来的模型也没法用来预测其他的图片，只能将训练时加入的测试集结果可视化。实在不是很明白为什么会造成现在这种情况，能否耽误您一点时间讲解一下
